src/test/java/test/listeners/ListenerTest.java

```diff
@@ -260,4 +267,31 @@ public void testFailATestFromOnTestStart() {
     Assert.assertEquals(adapter.getPassedTests().size(), 0);
     Assert.assertEquals(adapter.getFailedTests().size(), 1);
   }
+
+  @Test(dataProvider = "dp", description ="GITHUB-1029" )
+  public void ensureXmlTestIsNotNull(Class<?> clazz, XmlSuite.ParallelMode mode) {
+    XmlSuite xmlSuite = createXmlSuite("Suite");
+    createXmlTest(xmlSuite, "GITHUB-1029-Test", clazz);
+    xmlSuite.setParallel(mode);
+    Issue1029InvokedMethodListener listener = new Issue1029InvokedMethodListener();
+    TestNG testng = create(xmlSuite);
+    testng.addListener((ITestNGListener) listener);
+    testng.setThreadCount(10);
+    testng.setThreadCount(10);
```

Comment:
duplicate line

Reply:
Oops.. Copy paste error. Fixed and pushed.

Reply:
👍